client/src/main.rs

```diff
@@ -10,8 +10,6 @@ use std::io::{stdout, Read, Write};
 use std::sync::Arc;
 use tokio::sync::Mutex;
 
-const SSH_BUFFER_SIZE: usize = 2048;
-
```

Comment:
I thought that this should have gone much earlier.

Reply:
likely because it should have :P